src/main/java/com/internet/shop/controllers/bucket/AddItemToBucketController.java

```diff
@@ -0,0 +1,36 @@
+package com.internet.shop.controllers.bucket;
+
+import com.internet.shop.library.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.service.interfaces.ProductService;
+import com.internet.shop.service.interfaces.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/item/add/bucket")
+public class AddItemToBucketController extends HttpServlet {
```

Comment:
```suggestion
public class AddItemToShoppingCartController extends HttpServlet {
```
keep your naming consistent

Reply:
ok.